src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/LanguageServerTagHelperCompletionService.cs

```diff
@@ -169,7 +169,7 @@ public override ElementCompletionResult GetElementCompletions(ElementCompletionC
 
         AddAllowedChildrenCompletions(completionContext, elementCompletions);
 
-        if (elementCompletions.Count > 0)
+        if (elementCompletions.Any())
```

Comment:
> The one exception IMO would be the .Count > 0  to .Any() changes since I feel they're just more clear.

Ironically (or perhaps not? I don't know, ask Alanis) this is a change I would 100% prefer to be reverted, as the old code was better. `.Count` being accessed on a concrete type is always preferrable to using `.Any()` IMO

Reply:
You are likely correct, I mixed up `Count` and `Count()`.

Reply:
Ahh, that would make much more sense to me to change